doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -899,12 +899,12 @@ to keep your client decoupled from implementation details.
 
 # Group Entities
 
-### List entities [GET /v2/entities{?limit,offset,options,type,id,idPattern,q,georel,geometry,coords,attrs,orderBy}]
+### List entities [GET /v2/entities{?limit,offset,options,type,id,idPattern,typePattern,q,georel,geometry,coords,attrs,orderBy}]
 
-Retrieves a list of entities that match different criteria (by id, idPattern, type or those which
+Retrieves a list of entities that match different criteria (by id, idPattern, type, typePattern or those which
 match a query or geographical query). A given entity has to match all the criteria to be retrieved
 (i.e., the criteria is combined in a logical AND way). Note that the id and idPattern parameters are
```

Comment:
Probably better to say

"pattern matching query parameters are incompatible with their corresponding exact matching parameters, i.e. idPattern with `id` and `typePattern` with `type`.